irida_import/irida_import.py

```diff
@@ -628,6 +660,97 @@ def link(self, sample_file, folder_id):
 
         return added
 
+    def upload_file_to_galaxy(self, sample_file, folder_id):
+        """
+        Upload a sample file to Galaxy
+
+        :type sample_file: SampleFile
+        :param sample_file: the sample file to upload
+        :type folder_id: ID of folder to upload file to
+        :param sample_folder_path: the folder in Galaxy to store the file in
```

Comment:
should this actually be named `folder_id` instead of `sample_folder_path`, the function above has the same issue with the doc.

Reply:
Good catch. Updated in 97c187bc8ab0716b21f78c299f47c20ad4f1ae00